app/src/main/res/values/strings.xml

```diff
@@ -6,15 +6,15 @@
     <string name="position_not_found">Position konnte nicht ermittelt werden.</string>
     <string name="your_location">Sie sind hier.</string>
     <string name="show_map">Karte anzeigen</string>
-    <string name="description_sun">Die Sonne ist unsere Wärmequelle. Dank ihr ist es auf unserer Kugel temperaturmäßig auszuhalten.</string>
-    <string name="description_mercury">Merkur ist der am nähesten an der Sonne gelegene Planet.</string>
-    <string name="description_venus">Venus hat keine Bedeutung. Keiner interessiert sich für ihn. :D"</string>
-    <string name="description_earth">Die Erde. Unser Heimatplanet.</string>
-    <string name="description_mars">Hier laufen angeblich Marsmenschen rum.</string>
-    <string name="description_jupiter">Ein Gasplanet und derjenige, der in unserem Sonnensystem am ehesten eine Diät nötig hat.</string>
-    <string name="description_saturn">Woher hat Saturn eigentlich seine Ringe? So oft verheiratet?</string>
-    <string name="description_uranus">Mir fällt nichts zu Uranus ein. Pech gehabt.</string>
-    <string name="description_neptune">Da Pluto aus dem Sonnensystem geschmissen wurde, ist er jetzt ganz außen.</string>
+    <string name="description_sun">Die Sonne ist das Zentrum unseres Sonnensystems und gleichzeitig unsere Wärme- und Lichtquelle.</string>
+    <string name="description_mercury">Merkur ist der am nähesten an der Sonne gelegene und zugleich kleinste Planet unseres Sonnensystems.</string>
+    <string name="description_venus">Venus ist der der Erde am nächsten gelegene Planet und nachts nach dem Mond das zweithellste Himmelsgestirn."</string>
+    <string name="description_earth">Die Erde ist Ursprung und Heimat aller uns bisher bekannten Lebewesen.</string>
```

Comment:
Got you! Never watched independence day, did ya? I can't let you get away with this..